### PR TITLE
Subnavigation implementation

### PR DESCRIPTION
**Pull request by DarinaBlazevica**
- Add Subnavigation component
- Add styling
- Disable scroll for page if Menu or Submenu is open

### CODE REVIEW

File: src/components/molecules/Subnavigation/Subnavigation.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    subMenu: MenuData[]
+    backBtnMenu: () => void
+    path?: string;
+    title: string;
+}
+
+const Subnavigation = (props: SubnavigationProp) => {
+    const subnavigationProps = props
+
+    const reduceItem = subnavigationProps.subMenu.reduce<any>((subMenu, item) => {
```
Comment (ghost) [inline src/components/molecules/Subnavigation/Subnavigation.tsx:17]: Avoid any, let typescript infer subMenu type
Assert the initial value you provide to .reduce

**Review by ilze-dom — APPROVED** [review]

File: src/components/organisms/Topnavigation/Topnavigation.tsx [hunk 2]
```diff
@@ -52,8 +44,18 @@ const TopNavigation = (props: TopnavigationProps) => {
     };
   }, [Ref]);
 
+  {
+    isMenuOpen || isSubMenuOpen
```
Comment (ghost) [inline src/components/organisms/Topnavigation/Topnavigation.tsx:48]: Place this block(line 47-51) inside useEffect and add referenced variables to dependency array, we don't want to fire these on each render

**Review by ReinisZogots — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/components/molecules/Subnavigation/Subnavigation.tsx (hunk 1 above)
Comment (DarinaBlazevica) [reply src/components/molecules/Subnavigation/Subnavigation.tsx:17]: Solved

File: src/components/organisms/Topnavigation/Topnavigation.tsx (hunk 2 above)
Comment (DarinaBlazevica) [reply src/components/organisms/Topnavigation/Topnavigation.tsx:48]: Solved

**Pull request merged**